perftests/recv-zeros/Makefile.BSD

```diff
@@ -0,0 +1,39 @@
+# Program name.
+PROG	=	recv-zeros
+
+# Don't install it.
+NOINST	= 	1
+
+# Main test code
+SRCS	=	main.c
+
+# Data structures
+.PATH.c	:	../../libcperciva/datastruct
+SRCS	+=	elasticarray.c
+SRCS	+=	ptrheap.c
+SRCS	+=	timerqueue.c
+IDIRS	+=	-I../../libcperciva/datastruct
+
+# Event loop
```

Comment:
Are we using any of this libcperciva library code?

Reply:
whoops, that was left over from a much earlier version.

Reply:
same in send-zeros, sorry.